.circleci/config.yml

```diff
@@ -487,7 +471,9 @@ workflows:
             branches:
               only:
                 - raft-tdp-main
+
   build-and-test:
+    unless: << pipeline.parameters.run_dev_deployment >>
```

Comment:
There's checks that make sure this is completed first before running the dev-deployment workflow so there is no need to run this again when this parameter is set.